aws-rds-customdbengineversion/aws-rds-customdbengineversion.json

```diff
@@ -103,12 +115,12 @@
     "/properties/KMSKeyId": "$join([\"arn:(aws)[-]{0,1}[a-z]{0,2}[-]{0,1}[a-z]{0,3}:kms:[a-z]{2}[-]{1}[a-z]{3,10}[-]{0,1}[a-z]{0,10}[-]{1}[1-3]{1}:[0-9]{12}[:]{1}key\\/\", KMSKeyId])"
   },
   "required": [
-    "DatabaseInstallationFilesS3BucketName",
     "Engine",
     "EngineVersion"
   ],
   "writeOnlyProperties": [
-    "/properties/Manifest"
+    "/properties/Manifest",
+    "/properties/UseAwsProvidedLatestImage"
```

Comment:
/SourceCustomDbEngineVersionIdentifier is also not allowed for modification.

Reply:
Yes, those properties were added as `createOnlyProperties`. `writeOnlyProperties` are not returned as part of the describe and thus are exempt from drift detection. With `createOnlyProperties` it means changing this value will trigger a resource replacement as we can't modify it so we create a new resource with the new value.